code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -119,13 +119,22 @@ assumpF otherContents = SRS.assumpt (assumpIntro : otherContents) []
 
 -- | Wrapper for 'thModelIntro'. Takes the program name and other 'Contents'.
 thModF :: (Idea a) => a -> [Contents] -> Section
-thModF progName otherContents = SRS.thModel (thModIntro progName : otherContents) []
+thModF progName otherContents = if not (null otherContents) then
```

Comment:
You can use pattern matching on `otherContents` instead of this if-then-else.

Reply:
Ok, I will implement that right now!